docs/cloud/automation/terraform.md

```diff
@@ -1,2 +1,244 @@
+---
 
+# Terraform provider 
+
+## Installation
+
+The current version of the provider is : {{ $frontmatter.terraform_current_version }}.
+
+The releases are available in Terraform's official [registry][terraform registry] and via [GitHub releases][terraform github releases].
+
+### Terraform 0.13+
+
+Terraform supports third party modules installed via the plugin registry.  
+Add the following to your terraform module configuration.
+
+
+<<< @/docs/cloud/automation/snippets/providers_eventstore.tf.hcl
+
+### Terraform 0.12
+
+In order for Terraform to find the plugin, the appropriate binary must be placed into the Terraform third-party plugin directory.  
+The location varies by operating system:
+
+- Linux and macOS `~/.terraform.d/plugins`
+- Windows `%APPDATA%\terraform.d\plugins`
+
+Alternatively, the binary can be placed alongside the main `terraform` binary.
+
+You can download the provider using the following commands:
+
+- Linux:
+
+<<< @/docs/cloud/automation/snippets/download_provider_linux.sh
+ 
+- macOS:  
+
+<<< @/docs/cloud/automation/snippets/download_provider_macos.sh
+
+- Windows (PowerShell): 
+
+<<< @/docs/cloud/automation/snippets/download_provider_windows.ps1.powershell
+
+## From Source 
+
+If you prefer to install from source, use the `make install` target in this [repository][terraform github].
+
+You will need a Go 1.13+ development environment.
+
+## Provider Configuration
+
+The Event Store Cloud provider must be configured with an access token.   
+There are several additional options that may be useful.
+
+Provider configuration options are:
+
+| Option              | Environment Variable   | Description                                                                                                          |
+| :------------------ | :--------------------- | :------------------------------------------------------------------------------------------------------------------- |
+| `token`             | `ESC_TOKEN`            | *Required*, your access token for Event Store Cloud                                                                  |
+| `organization_id`   | `ESC_ORG_ID`           | *Required*, the identifier of your  Event Store Cloud organization.                                                  |
+| `url`               | `ESC_URL`              | *Optional*, the URL of the Event Store Cloud API. This defaults to the public cloud instance of Event Store Cloud    |
+| `token_store`       | `ESC_TOKEN_STORE`      | *Optional*, the location on the local filesystem of the token cache. This is shared with the Event Store Cloud CLI.  |
+                                                                                                                                                                       
+### Obtaining the `token`
+Currently this token must be created and displayed with the [esc cli][esc cli github releases] tool.  
+
+The token id displayed in the cloud console is not a valid token.
```

Comment:
WOT? o_O Shouldn't we fix that on the Cloud UI?

Reply:
I think it has been fixed now `Console token creation now functions correctly in all browsers.` Needs checking though

Reply:
is it when you create a new token in the console ?
what browser did not work ?